models/POP/shell_scripts/cesm2_1/CESM2_1_setup_pmo

```diff
@@ -0,0 +1,763 @@
+#                 configuration files will be stored.  This should probably not
+#                 be in scratch (on GLADE, your 'work' partition is suggested).
+#                 This script will delete any existing CASEROOT, so this script,
+#                 and other useful things should be kept elsewhere.
+# RUNDIR          Will create the CESM run directory here.  Will need large
+#                 amounts of disk space, generally on a scratch partition.
+# EXEROOT         Will create the CESM executable directory here, where the
+#                 CESM executables will be built.  Medium amount of space
+#                 needed, generally on a scratch partition.
+# ARCHDIR         Will create the CESM short-term archive directories here.
+#                 Large, generally on a scratch partition.  Files will remain
+#                 here until the long-term archiver moves it to permanent storage.
+# ==============================================================================
+
+setenv CASEROOT     /glade/work/${USER}/cases/${CASE}
+setenv RUNDIR       /glade/scratch/${USER}/${CASE}/run
+setenv EXEROOT      /glade/scratch/${USER}/${CASE}/bld
+setenv ARCHDIR      /glade/scratch/${USER}/archive/${CASE}
+
+# ==============================================================================
+# Create the case - this creates the CASEROOT directory.
+#
+# For list of the pre-defined component sets: ./create_newcase --list
+# To create a variant compset, see the CESM documentation and carefully
+# incorporate any needed changes into this script.
+#
+# The script won't allow the user to make CASEROOT the same dir as where this
+# setup script is since the build process removes all files in the ${CASEROOT}
+# dir before populating it.
+# ==============================================================================
+
+if ( $CASEROOT == `dirname $0` ) then
+   echo "ERROR: the setup script should not be located in the CASEROOT"
+   echo "directory, because all files in the CASEROOT dir will be removed"
+   echo "before creating the new case.  move the script to a safer place."
+   exit -1
+endif
+
+echo "removing old files from ${CASEROOT}"
+echo "removing old files from ${EXEROOT}"
+echo "removing old files from ${RUNDIR}"
+${REMOVE} ${CASEROOT}
+${REMOVE} ${EXEROOT}
+${REMOVE} ${RUNDIR}
+
+# ==============================================================================
+# Invoking create_newcase.
+# The --run-unsupported option is necessary for CESM2 because this compset
+# hasn't been tested.
+# ==============================================================================
+
+set use_tasks_per_node = 36
+set nthreads = 1
+
+echo ''
+echo 'Invoking create_newcase.'
+echo ''
+echo '${cesmroot}/cime/scripts/create_newcase'
+echo '--case ${CASEROOT}'
+echo '--mach ${mach}'
+echo '--res ${resolution}'
+echo '--compset ${compset}'
+echo '--run-unsupported'
+echo '--pecount ${use_tasks_per_node}x${nthreads}'
```

Comment:
The single quotes prevent the expansion of the variable names. You must use double quotes.
This happens in multiple scripts,.